src/elementPlugins/dropdownChoice/src/SummaryView.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { Header, Icon } from 'semantic-ui-react'
+import { SummaryViewProps } from '../../types'
+
+const SummaryView: React.FC<SummaryViewProps> = ({ parameters, value }) => {
+  return (
+    <>
```

Comment:
Better to use `Container` or `Segment.Group` here

Reply:
Again, both those components stylise the element in their own box, which is not what we want for the Summary page (according to current designs)